study.md

```diff
@@ -20,21 +20,20 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
-```
+The event loop is what passes setTimeOut functions into the stack after the web API has ran it's set time and once the stack is freed up.```
```

Comment:
So the event loop is what all events go through, i think you're confused because that video used setTimeOut functions to demonstrate this.